lib/pdk/telemetry/client/google_analytics.rb

```diff
@@ -0,0 +1,132 @@
+module PDK
+  module Telemetry
+    module Client
+      class GoogleAnalytics
+        PROTOCOL_VERSION  = 1
+        APPLICATION_NAME  = 'pdk'.freeze
+        TRACKING_ID       = 'UA-xxxx-1'.freeze
+        TRACKING_URL      = 'https://google-analytics.com/collect'.freeze
+        CUSTOM_DIMENSIONS = {
+          operating_system: :cd1,
+          inventory_nodes:  :cd2,
+          inventory_groups: :cd3,
```

Comment:
Can probably remove this bolt specific custom dimensions

Reply:
Yeah, I was going to leave that to be handled in a follow up PR, but for now I've removed the dimensions that we definitely won't be using.